Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -462,11 +462,10 @@ to the PID parameters.
 </li>
 <li>
 The autotuning must be conducted when the process is in a stable state.
-The user should monitor changes in the independent variables and
-the controlled variables (e.g., mass flow rate, temperature) over time.
-When the changes in the independent variables are small (e.g., less than 10%),
-and the changes in the controlled variables are either small or
-exhibit regular oscillations,
+The user should monitor changes in the disturbances of the control plant, e.g.,
+outdoor drybulb temperature, and the controller output <code>y</code> over time.
+When the changes in the independent variables are small (e.g., less than 10%) and the
```

Comment:
still using the `independent variables` here, may change it to the `inputs and outputs`.

Reply:
I replace the <code>independent variables</code> with <code>disturbances</code>